cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/loki/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - lokistack-gateway-auth-delegator.yaml
+  - lokistack-tenant-logs.yaml
```

Comment:
Please use the pattern we always use and don't create new ones. Read this readme please:

https://github.com/operate-first/apps/tree/master/cluster-scope#application-folder-structure

Each resource has it's own folder. If you need to bundle resources together, you can create new bundle in `clsuter-scope/bundles`.